src/ElectricPokemon.java

```diff
@@ -0,0 +1,43 @@
+import java.util.Arrays;
+import java.util.List;
+
+public class ElectricPokemon  extends Pokemon {
+
+    private String food = "volts";
+    private List<String> attacks = Arrays.asList("thunderpunch","electroball","thunder","volttackle");
+
+    public ElectricPokemon(String name) {
+        super("electric");
```

Comment:
Mischien is het handig om een Pokemon constructor te hebben waar zowel naam als type wordt gevraagd